components/GroupSingleQ.js

```diff
@@ -1,123 +1,67 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Dimensions, StyleSheet, Text, View } from 'react-native';
-import { Body, Card, CardItem, Input, Item } from 'native-base';
+import { Body, Button, Card, CardItem, Input, Item } from 'native-base';
 import { CheckBox } from 'react-native-elements';
 
 const height = Dimensions.get('window').height;
 
-export default function GroupSingleQ({ question }) {
+export default function GroupSingleQ({ question, answerQuestion }) {
+
+  const [checkWho, setCheckWho] = useState([]);
+
+  const questions = question.question.items.map((item, questionIndex) => {
+    return <CardItem key={questionIndex} style={styles.questionCardItem}>
+      <Body>
+        <Text style={styles.questionText}>
+          {item.name}
+        </Text>
+        <View style={styles.checkboxes}>
+          {item.choices.map((choice, checkBoxIndex) => {
+            console.log(checkWho)
```

Comment:
Oh shoot, we'll have to clean this up as well. But this is great. Thank you so much for adding the map logic, it is SO much cleaner!